readthedocs/search/documents.py

```diff
@@ -246,16 +250,16 @@ def prepare_publisher_project(self, instance):
         """Prepare docsitalia publisher project field."""
         # not using more sophisticated Django methods in order to exploit prefetching
         try:
-            return instance.project.publisherproject_set.all()[0].slug
-        except IndexError:
+            return instance.project.publisherproject_set.first().slug
+        except AttributeError:
             return
 
     def prepare_publisher(self, instance):
         """Prepare docsitalia publisher field."""
         # not using more sophisticated Django methods in order to exploit prefetching
         try:
-            return instance.project.publisherproject_set.all()[0].publisher.name
-        except IndexError:
+            return instance.project.publisherproject_set.first().publisher.name
+        except AttributeError:
```

Comment:
Questa modifica forse è slegata dalla issue e introduce un disallineamento non necessario dal codice master di RTD complicandoci il compito in futuro di allinearci quindi direi di ripristinare la versione precedente.

Reply:
è codice custom nostro :)